apps/rewards/app/utils/token-utils.js

```diff
@@ -73,3 +74,11 @@ export async function getTokenName(app, address) {
 
   return tokenName ? toUtf8(tokenName) : null
 }
+
+export async function getTokenStartBlock(app, address) {
```

Comment:
I think you can just import this here https://github.com/AutarkLabs/planning-suite/blob/dev/apps/rewards/app/store/app.js

Reply:
utils are normally imported straight from their file so it's easier to trace them.